test/fixtures/jquery-ui-1.12.1/base/theme.css

```diff
@@ -1,5 +1,5 @@
 /*!
- * jQuery UI CSS Framework @VERSION
+ * jQuery UI CSS Framework 1.12.1
```

Comment:
For the fixtures, I just took `base/theme.css` from https://github.com/jquery/jquery-ui/blob/1.13.2/themes/base/theme.css and `themes/smoothness.css` from a proper bundle from https://jqueryui.com/download/. The only change I had to make in that latter file is updating the URL after "To view and modify this theme, visit ..." since it's not stable.

I did the same for `1.12.1`.